must-gather/collection-scripts/namespace

```diff
@@ -22,8 +22,5 @@ function nro_namespace() {
   # namespace suggested by the documentation. This is a fancier way to check for its existence
   [ -z "${ns}" ] && ns=$(oc get ns openshift-numaresources -o=jsonpath='{.metadata.name}{"\n"}' 2> /dev/null )
 
-  # we should never get there. This is the last resort.
-  [ -z "${ns}" ] && ns="openshift-numaresources"
-
```

Comment:
why delete this?

Reply:
This is the way we find out the NRO namespace is not defined.
We were using a "default" when everything else fails but if we cannot find the namespace and we reach this point is good to assume NRO is not installed.

Reply:
makes sense, thanks for clarifying.